app/assets/stylesheets/helpers/_contents-list.scss

```diff
@@ -0,0 +1,91 @@
+.contents-list {
+  margin-bottom: $gutter-two-thirds;
+
+  // Always render the contents list above a
+  // back to contents link
+  position: relative;
+  z-index: 1;
+  background: $white;
+  box-shadow: 0 20px 15px -10px $white;
+
+  li {
+    padding-top: $gutter / 4;
+    line-height: 1.3;
+    list-style-type: none;
+  }
+
+  // Remove underlines from lists of links
+  // to improve legibility
+  a {
+    text-decoration: none;
+
+    &:hover,
+    &:focus,
+    &:active {
+      text-decoration: underline;
+    }
+  }
+
+  &.contents-list-nested,
+  &.contents-list-dashed {
+    li {
+      margin-left: $gutter-half + 5;
+      padding-right: $gutter-half + 5;
+      list-style-type: disc;
+
+      @supports (content: "—") {
+        list-style-type: none;
+
+        &:before {
+          content: "— ";
+          position: relative;
+          float: left;
+          width: $gutter-half + 5;
+          margin-left: -($gutter-half + 5);
+        }
+      }
+    }
+  }
+
+  // When nesting contents lists, the top level items have no
+  // list marker and are styled in bold
+  &.contents-list-nested {
+    .nested-contents-item-h2 {
+      margin-left: 0;
+      list-style-type: none;
```

Comment:
Not sure this needs to be defined here - the main styles for this list already set list-style-type to none.